### PR TITLE
Update dependencies v2

### PR DESCRIPTION
**Pull request by Grekz**

### CODE REVIEW

File: src/atoms/ListItem/listItem.stories.tsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import ListItem from '.'
 storiesOf('Atoms|ListItem', module).add('As Default', () => (
   <>
     <ListItem>List item</ListItem>
-    <ListItem isActive>Active list item</ListItem>
+    <ListItem isActive={true}>Active list item</ListItem>
```
Comment (adancarrasco) [inline src/atoms/ListItem/listItem.stories.tsx]: Why are we passing an implicit value here?

File: src/molecules/ButtonGroup/buttonGroup.component.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import styled from '@emotion/styled'
 import * as React from 'react'
 
-import ButtonType from '../../atoms/Button/button.component'
+import { ButtonType } from '../../atoms/Button'
```
Comment (adancarrasco) [inline src/molecules/ButtonGroup/buttonGroup.component.tsx]: 👍

File: src/molecules/List/list.stories.tsx [hunk 3]
```diff
@@ -10,6 +10,6 @@ storiesOf('Molecules|List', module).add('As Default', () => (
   <List>
     <ListItem>List item</ListItem>
     <ListItem>List item</ListItem>
-    <ListItem isActive>Active list item</ListItem>
+    <ListItem isActive={true}>Active list item</ListItem>
```
Comment (adancarrasco) [inline src/molecules/List/list.stories.tsx]: Same as above about the implicit value.

File: src/atoms/ListItem/listItem.stories.tsx (hunk 1 above)
Comment (Grekz) [reply src/atoms/ListItem/listItem.stories.tsx]: Umm, I think this was update by a new prettier rule =)

File: src/molecules/List/list.stories.tsx (hunk 3 above)
Comment (Grekz) [reply src/molecules/List/list.stories.tsx]: Same answer

**Comment by adancarrasco** [comment]
LGTM. 👍

**Pull request merged**